tests/Multiple/MediumMultiplePlainTextFormatterTest.php

```diff
@@ -187,7 +187,7 @@ public function testItWillShowEventHasConcludedWhenAllPastDatesAreHidden(): void
 
         $event = $event->withSubEvents($newEvents);
 
-        $expectedOutput = 'Dit evenement is afgelopen';
+        $expectedOutput = 'evenement afgelopen';
```

Comment:
Can we still start with a capital letter though? That would keep it consistent with the other summaries.